lib/setup-lib.bash

```diff
@@ -100,37 +87,32 @@ function asdf_bin_in_path() {
 }
 
 function installed_direnv() {
-  local bin
-  bin="${ASDF_DIRENV_BIN:-$(asdf which direnv 2>/dev/null || true)}"
-  if test -x "$bin"; then
-    ASDF_DIRENV_BIN="$bin"
-  else
-    local version
-    version="$(prompt "Please specify a direnv version to use [version-number/latest/SYSTEM]" "system")"
-    case "$version" in
-      system | SYSTEM | "")
-        ASDF_DIRENV_BIN="$(run_cmd env ASDF_DIRENV_VERSION=system asdf which direnv)"
-        ;;
-      latest | LATEST)
-        run_cmd asdf install direnv latest
-        version="$(asdf list direnv | tail -n 1 | sed -e 's/ //g')" # since `ASDF_DIRENV_VERSION=latest asdf which direnv` does not work
-        ASDF_DIRENV_BIN="$(run_cmd env ASDF_DIRENV_VERSION="$version" asdf which direnv)"
+  local version=$1
+  case "$version" in
+    system | SYSTEM | "")
```

Comment:
`$version` is required in the code bellow, I guess we can remove the empty case `""` ?

Reply:
Removed!